gettsim/transfers/grunds_im_alter.py

```diff
@@ -0,0 +1,445 @@
+    ).clip(lower=0)
+
+    # Calculate and return monthly capital income (after deduction)
+    return capital_income_y / 12
+
+
+def grunds_im_alter_priv_rente_m(
+    priv_rente_m: FloatSeries,
+    arbeitsl_geld_2_params: dict,
+    grunds_im_alter_params: dict,
+) -> FloatSeries:
+    """Calculate private pension benefits which are considered in the calculation of
+    Grundsicherung im Alter.
+
+    Legal reference: § 82 SGB XII Abs. 4
+
+    Parameters
+    ----------
+    priv_rente_m
+        See basic input variable :ref:`priv_rente_m <priv_rente_m>`.
+    arbeitsl_geld_2_params
+        See params documentation :ref:`arbeitsl_geld_2_params
+        <arbeitsl_geld_2_params>`.
+    grunds_im_alter_params
+        See params documentation :ref:`grunds_im_alter_params <grunds_im_alter_params>`.
+
+    Returns
+    -------
+
+    """
+    priv_rente_m_amount_exempt = piecewise_polynomial(
+        x=priv_rente_m,
+        thresholds=grunds_im_alter_params["priv_rente_anr_frei"]["thresholds"],
+        rates=grunds_im_alter_params["priv_rente_anr_frei"]["rates"],
+        intercepts_at_lower_thresholds=grunds_im_alter_params["priv_rente_anr_frei"][
+            "intercepts_at_lower_thresholds"
+        ],
+    )
+
+    priv_rente_m_amount_exempt = priv_rente_m_amount_exempt.clip(
+        upper=arbeitsl_geld_2_params["regelsatz"][1] / 2
+    )
+
+    return priv_rente_m - priv_rente_m_amount_exempt
+
+
+def _grunds_im_alter_mehrbedarf_behinderung_m_hh(
+    _grunds_im_alter_mehrbedarf_behinderung_m: FloatSeries, hh_id: IntSeries,
+) -> FloatSeries:
+    """Aggregate mehrbedarf for people with disabilities on household level.
+
+    Parameters
+    ----------
+    _grunds_im_alter_mehrbedarf_behinderung_m
+        See :func:`_grunds_im_alter_mehrbedarf_behinderung_m`.
+    hh_id
+        See basic input variable :ref:`hh_id <hh_id>`.
+    Returns
+    -------
+
+    """
+    return _grunds_im_alter_mehrbedarf_behinderung_m.groupby(hh_id).sum()
+
+
+def _grunds_im_alter_mehrbedarf_behinderung_m(
+    schwerbeh_ausweis_g: BoolSeries,
+    hhsize_tu: IntSeries,
+    grunds_im_alter_params: dict,
+    arbeitsl_geld_2_params: dict,
+    tu_id: IntSeries,
+) -> FloatSeries:
+    """Calculate mehrbedarf for people with disabilities.
+
+    Parameters
+    ----------
+    schwerbeh_ausweis_g
+        See basic input variable :ref:`behinderungsgrad <schwerbeh_ausweis_g>`.
+    hhsize_tu
+        See :func:`hhsize_tu`.
+    ges_rentenv_params
+        See params documentation :ref:`ges_rentenv_params <ges_rentenv_params>`.
+    arbeitsl_geld_2_params
+        See params documentation :ref:`arbeitsl_geld_2_params <arbeitsl_geld_2_params>`.
+    tu_id
+        See basic input variable :ref:`tu_id <tu_id>`.
+    Returns
+    -------
+
+    """
+    out = pd.Series(0, index=schwerbeh_ausweis_g.index, dtype=float)
+    hhsize_tu = tu_id.replace(hhsize_tu)
+
+    # mehrbedarf for disabilities = % of regelsatz of the person getting the mehrbedarf
```

Comment:
So this is confusing. Are we talking only about people with `schwerbeh_ausweis_g` (impaired in movement, eysight, etc. AFAIU) or anyone with some registered disability? Adjust comment appropriately.

Reply:
Currently only Mehrbedarf for Ausweis G individuals is implemented.

@paulinaschroeder, do you know if there is any "Mehrbedarf" for individuals with disabilities, but no `schwerbeh_ausweis_g`?

Reply:
No there isn’t. You have to have a Schwerbehindertenausweis mit Merkzeichen G to qualify as someone with a disability.
The only exception is if the disability requires a special diet (e.g. diabetes), then you might qualify for a different Mehrbedarf to compensate those special needs.

Reply:
Thanks! Then the adjusted docstring `Calculate mehrbedarf for individuals with disabled person's pass G.` is sufficient.

Reply:
And the function name should become:

`_grunds_im_alter_mehrbedarf_schwerbeh_g_m` 

(inserting "schwer")

Thanks!